lib/rpm_package.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (C) IBM Corp. 2016.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import logging
+import yaml
+
+from lib import config
+from lib import exception
+from lib.package import Package
+
+CONF = config.get_config().CONF
+LOG = logging.getLogger(__name__)
+
+
+class RPM_Package(Package):
+
+    def __init__(self, name, distro, *args, **kwargs):
+        super(RPM_Package, self).__init__(name, *args, **kwargs)
+        self.distro = distro
+
+    def load(self):
+        """
+        Read yaml file describing this package.
+        Optionally, download its source code.
+        """
+        super(RPM_Package, self).load()
+        try:
+            with open(self.package_file, 'r') as package_file:
+                package_info = yaml.load(package_file).get('Package')
+        except IOError:
+            raise exception.PackageDescriptorError(
+                "Failed to open %s's YAML descriptor" % self.name)
```

Comment:
That's better indeed.
I've renamed package_info to package_data and added it as an object attribute (self.package_data).